README.adoc

```diff
@@ -226,16 +225,16 @@ Now, set a fallback method to deal with this failure.
 ----
 
 InventoryManager.java
-[source, java, linenums, role='code_column']
+[source, java, linenums, role='code_column hide_tags=copyright tags=add_fallback']
```

Comment:
- don't think `tags=add_fallback` is necessary
- also extra blank line appearing on top of file in codepane

Reply:
addressed.